src/solidlsp/language_servers/yaml_language_server.py

```diff
@@ -0,0 +1,196 @@
+"""
```

Comment:
pls write tests ensuring that basic stuff works, like for all other LS that we support

Reply:
done